src/components/person.js

```diff
@@ -3,23 +3,30 @@ import PeopleActions from '../actions/people_actions';
 import ClassNames from 'classnames';
 
 class Person extends React.Component {
-  handleClick() {
+  remove() {
     PeopleActions.remove(this.props.index);
   }
 
+  select() {
+    PeopleActions.selectDriver(this.props.index);
+  }
+
   render() {
     var classes = ClassNames({
       active: this.props.isCurrent
     });
 
     return (
-      <li className={classes}>
-        <a href="#" className="remove" onClick={this.handleClick.bind(this)}>x</a>
+      <li className={classes} onClick={this.select.bind(this)}>
+        <a href="#" className="remove" onClick={event => {
+          event.stopPropagation()
```

Comment:
We have to stop propagation, or else the click will trigger the `li`'s click event, thus calling the "select driver" action.